package.json

```diff
@@ -15,5 +23,46 @@
   "bugs": {
     "url": "https://github.com/rackerlabs/helix-react/issues"
   },
-  "homepage": "https://github.com/rackerlabs/helix-react#readme"
+  "homepage": "https://github.com/rackerlabs/helix-react#readme",
+  "dependencies": {
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.5.4",
+    "react": "^15.0.0 || ^16.0.0",
+    "react-dom": "^15.0.0 || ^16.0.0",
```

Comment:
I think we can update the version to `>=15` (based on testing at [https://semver.npmjs.com/]).

```javascript
"peerDependencies": {
    ...
    "react": ">=15",
    "react-dom": ">=15"
    ...
}
```